x-pack/plugins/canvas/canvas_plugin_src/renderers/image.tsx

```diff
@@ -19,7 +22,8 @@ export const image: RendererFactory<Arguments> = () => ({
   displayName: strings.getDisplayName(),
   help: strings.getHelpDescription(),
   reuseDomNode: true,
-  render(domNode, config, handlers) {
+  render: async (domNode, config, handlers) => {
+    const { elasticLogo } = await getElasticLogo();
```

Comment:
Same as mentioned above, if not a valid url throw an error instead of requiring the elastic outline?

Reply:
Done.